clang/lib/Driver/ToolChains/Arch/RISCV.cpp

```diff
@@ -142,6 +142,13 @@ void riscv::getRISCVTargetFeatures(const Driver &D, const llvm::Triple &Triple,
     Features.push_back("-relax");
   }
 
+  // -mfence-tso is default, unless -mno-fence-tso is specified.
+  if (Args.hasFlag(options::OPT_mfence_tso, options::OPT_mno_fence_tso, true)) {
+    Features.push_back("-no-fence-tso");
```

Comment:
If the -mfence-tso is the default in driver and backend, we should avoid `-no-fence-tso`, making the default case command line shorter.